wrapper/src/test/java/integration/refactored/container/tests/AuroraFailoverTest.java

```diff
@@ -571,45 +461,6 @@ public void test_takeOverConnectionProperties() throws SQLException, Interrupted
     }
   }
 
-  @TestTemplate
-  @EnableOnTestFeature(TestEnvironmentFeatures.NETWORK_OUTAGES_ENABLED)
-  public void test_failoverTimeoutMs() throws SQLException {
```

Comment:
Do we have equivalent unit test that checks failover timeout?

Reply:
[ClusterAwareReaderFailoverHandlerTest#testFailover_timeout ](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/wrapper/src/test/java/software/amazon/jdbc/plugin/failover/ClusterAwareReaderFailoverHandlerTest.java#L121) and https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/7b004974c994854b8e066b0f65a891ba205d63bf/wrapper/src/test/java/software/amazon/jdbc/plugin/failover/ClusterAwareWriterFailoverHandlerTest.java#L300